src/NzbDrone.Core/ImportLists/AniList/List/AniListParser.cs

```diff
@@ -0,0 +1,146 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using NLog;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.ImportLists.Exceptions;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.ImportLists.AniList.List
+{
+    public class AniListParser : IParseImportListResponse
+    {
+        private readonly Dictionary<int, MediaMapping> _mappings;
+        private readonly Logger _logger;
+        private readonly AniListSettings _settings;
+
+        public AniListParser(Logger logger, AniListSettings settings, Dictionary<int, MediaMapping> mappings)
```

Comment:
Nit: Logger last

Reply:
The reason logger was first here, was because it is a dependency from the container, whereas the setting and mapping objects are not. Typically dependency injection parameters are ordered first, even though it is manually passed in here.

Please confirm.

Reply:
Generally logger is one of the last parameters passed in. Though since the parser is not being constructed through DI we shouldn't be passing in the logger from `AniListImport`, instead we should instantiate the logger in the class, like `SimklParser` does.

Reply:
Refactored as recommended.